examples/species/animated-mice.js

```diff
@@ -0,0 +1,114 @@
+// TODO: This file was created by bulk-decaffeinate.
+// Sanity-check the conversion and remove this comment.
+/*
+ * decaffeinate suggestions:
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+*/
+
+class Mouse extends Populations.helpers.mixOf(Populations.BasicAnimal, Populations.AnimatedAgent) {
+  step() {
+    return this.wander();
+  }
+
+  wander(speed){
+    if (speed == null) {
+      const defaultSpeed = this.get('default speed');
+      speed = this.get('speed');
+      speed += (Math.random() * 2 - 1) * .1;
+      speed = Math.min(Math.max(speed, defaultSpeed * .75), defaultSpeed * 1.25);
+      this.set('speed', speed);
```

Comment:
Trying to keep the speed within 25% of the initial default range.  If we vary too far, the movement looks out of sync with the animation frame rate.